scanpipe/models.py

```diff
@@ -562,6 +562,14 @@ class Project(UUIDPKModel, ExtraDataFieldMixin, UpdateMixin, models.Model):
     notes = models.TextField(blank=True)
     settings = models.JSONField(default=dict, blank=True)
     labels = TaggableManager(through=UUIDTaggedItem)
+    purl = models.CharField(
+        max_length=2048,
+        blank=True,
+        help_text=_(
+            "Package URL for the project, used for pushing project scan result to "
+            "FederatedCode. This should be the PURL of the input."
```

Comment:
> This should be the PURL of the input.

Could you clarify what this means?

Reply:
> Could you clarify what this means?

Suppose we have https://registry.npmjs.org/lodash/-/lodash-4.17.21.tgz as the input, then the PURL should be pkg:npm/lodash@4.17.21.

Reply:
What about:
```
help_text=_(
    "Package URL (PURL) for the project, required for pushing the project's scan result to FederatedCode. "
    "For example, if the input is an input URL like https://registry.npmjs.org/lodash/-/lodash-4.17.21.tgz, "
    "the corresponding PURL would be pkg:npm/lodash@4.17.21."
)
```